common/database/database_update_manifest.cpp

```diff
@@ -5016,7 +5016,8 @@ CREATE TABLE `spawn2_disabled` (
 ) ENGINE=InnoDB AUTO_INCREMENT=1 DEFAULT CHARSET=utf8mb4;
 INSERT INTO spawn2_disabled (spawn2_id, disabled) SELECT id, 1 FROM spawn2 WHERE enabled = 0;
 ALTER TABLE `spawn2` DROP COLUMN `enabled`;
-)"
+)",
+		.content_schema_update = true
```

Comment:
Wasn't this both a content and state update with `spawn2_disabled` being a state table?

Reply:
Yep - adjusted. Not going to much matter for 99% of folks - I just retroactively did a handful for examples going forward. In the future we'd want updates split